### PR TITLE
Add 1.4.21 Release post

### PR DESCRIPTION
**Pull request by Nightfirecat**

### CODE REVIEW

**Comment by Adam-** [comment]
The three step clue image is 100x bigger than all of the other images?

File: src/_posts/2018-10-11-10-00-1.4.21-Release.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+[@Eadgars-ruse](https://github.com/eadgars-ruse) for submitting
+[(and demonstrating in video form)](https://streamable.com/ujaar) this feature!
+
+![Master-level three step cryptic clue solution](/public/img/blog/1.4.21-Release/master-three-step-cryptic-clue-full.png)
+![Partial solution for master-level three step cryptic clue](/public/img/blog/1.4.21-Release/master-three-step-cryptic-clue-partial.png)
+
+## Grand Exchange partial name searching
+
+The Grand Exchange search panel now allows for partial name searching, yielding better results than
+before. Thanks to [@Adam-](https://github.com/Adam-) for implementing this feature.
```
Comment (Adam-) [inline src/_posts/2018-10-11-10-00-1.4.21-Release.md]: Also `!price`

**Comment by Nightfirecat** [comment]
The 3-step clue images look fine to me?

https://github.com/runelite/runelite.net/blob/579d6551a2f3e983a71a3ce8b5b417193b3d7a89/public/img/blog/1.4.21-Release/master-three-step-cryptic-clue-full.png
https://github.com/runelite/runelite.net/blob/579d6551a2f3e983a71a3ce8b5b417193b3d7a89/public/img/blog/1.4.21-Release/master-three-step-cryptic-clue-partial.png

File: src/_posts/2018-10-11-10-00-1.4.21-Release.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+![XP globes default skill colors](/public/img/blog/1.4.21-Release/xp-globes-default-skill-color.png)
+![Updated XP globes progress overlay](/public/img/blog/1.4.21-Release/xp-globes-progress-overlay.png)
+![2005 interface chat background](/public/img/blog/1.4.21-Release/2005-chat-background.gif)
+
+There are also several smaller improvements and bug fixes, including:
+
+- The percentage completion component of the XP tracker is now accurate to two decimal places
+- The 2010 interface style has had some tab icon offsets updated to be more accurate to the style
+- Cavaliers will now display their buy limit in the grand exchange plugin
+- Chat messages which have been modified will no longer remain in that modified state once cycled
```
Comment (Adam-) [inline src/_posts/2018-10-11-10-00-1.4.21-Release.md]: This is probably too technical to include here

**Comment by Nightfirecat** [comment]
Updated, and everything's looking good now imo.

**Comment by trimbe** [comment]
I don't think it's necessary to have the gifs explaining the use of the tabs in the post. IMO they should just be put in the wiki and then linked to from the post.

**Comment by Adam-** [comment]
I agree.

**Comment by Nightfirecat** [comment]
I'll work on cutting that section down in size then.

**Comment by Nightfirecat** [comment]
Moved the gifs to the wiki page and linked it from the bank tag tabs section.

**Pull request merged**